### ISSUE TITLE
Frequent warnings "Communication failed during replication"

### ISSUE THREAD

**Issue by jakirkham** [dask/distributed#2188]
Am seeing this when using Distributed (no-backends) with spawn and Adaptive. The warning continues to occur even after the computation has completed successfully.

**Comment by mrocklin** [comment]
Indeed, that does sound frustrating.  If anyone has time to investigate the `Scheduler.replicate` method it sounds like that would be useful.

https://github.com/dask/distributed/blob/cb10d6b10ad7800648b44ab3b194220d1e6fc5f8/distributed/scheduler.py#L2463-L2559

**Comment by jakirkham** [comment]
FWIW disabling Adaptive and manually scaling fixed the issue.
